aldryn_forms/cms_plugins.py

```diff
@@ -494,6 +496,39 @@ def form_pre_save(self, instance, form, request, **kwargs):
         form.cleaned_data[field_name] = filer_file
 
 
+class ImageField(FileField):
+    name = _('Image upload field')
+
+    model = models.ImageUploadFieldPlugin
+
+    form = ImageFieldForm
+    form_field = RestrictedImageField
+    form_field_widget = RestrictedImageField.widget
+    fieldset_general_fields = Field.fieldset_general_fields + [
+        'upload_to',
```

Comment:
It's the folder where the files fill be uploaded to. I take care to set `is_public` to false for those files. Is that enough?